docker/README.md

```diff
@@ -43,110 +43,161 @@ See all available tags [here](https://hub.docker.com/repository/docker/bentoml/b
 
 Before starting:
 
+* We will utilize `docker buildx` underthe hood, so make sure to have that installed with `docker buildx install` (buildx should already installed for folks with Docker > 19.03)
```

Comment:
Should we require this anywhere that's not the CI? Can we make it so that locally we can just use `docker build` and then use `buildx` to create multiarch images when running CD? I feel like that would limit the complexity that users that just want to tool around with the dockerfile a little bit are exposed to.

We can still have all the fun `manager` stuff but it doesn't seem to me like we should expose most of it to the average user.

Reply:
users can still build the docker images per each dockerfile, but for us we will use buildx to support multiple architecture

Reply:
Sure, I mean that we don't need to expose this much complexity in the README, then. I'd prefer if `buildx` was only really necessary for building on CD, but I need to really review everything first.